src/main/webapp/app/exercises/programming/manage/grading/tasks/programming-exercise-grading-tasks-table.component.html

```diff
@@ -93,7 +93,19 @@ <h2 class="fw-medium">Tasks</h2>
         <button
             type="button"
             class="task__row__cell sort__button"
-            [ngbTooltip]="'artemisApp.programmingExercise.configureGrading.help.visibility' | artemisTranslate"
+            [ngbTooltip]="
+                this.isExamExercise
+                    ? ('artemisApp.programmingExercise.configureGrading.help.visibilityExam'
+                      | artemisTranslate
+                          : {
+                                afterReleaseDateOfResults: 'artemisApp.programmingExerciseTestCase.visibility.AfterReleaseDateOfResults' | artemisTranslate
+                            })
+                    : ('artemisApp.programmingExercise.configureGrading.help.visibility'
+                      | artemisTranslate
+                          : {
+                                afterDueDate: 'artemisApp.programmingExerciseTestCase.visibility.AfterDueDate' | artemisTranslate
+                            })
+            "
```

Comment:
Maybe this could be more readable if you only have one transaation pipe after the selection at the end?

Reply:
My understanding of the use of pipes does not seem to be sufficient here, could you supply a code suggestion of how you would imagine it? 

For my understanding the pipe in line 99 is needed to pass parameters to the translation, and in 101 still the placeholder needs to be translated, so there is the need for a second usage of the pipe.
The same applies for 103-107. 

Is there an easier way to solve it?

Reply:
Oh I did not notice the placeholder before, but thought that all of these were different cases. Is the placeholder really needed? Could you maybe integrate the string directly into the translation?

Reply:
My intention behind the placeholders was to ensure the translation of a `Visibility-Option` is changed and that it is consistent in the description.

But the translation itself can be changed, of course, as it looks like
```
"visibility": "Feedback und Punkte für Testfälle, die mit „{{ afterDueDate }}“ markiert sind, werden erst danach für Studierende angezeigt. Nie sichtbare Tests werden nicht in die Punktebewertung einbezogen.",
"visibilityExam": "Feedback und Punkte für Testfälle, die mit „{{ afterReleaseDateOfResults }}“ markiert sind, werden erst danach nach der Veröffentlichung der Klausurergebnisse angezeigt. Nie sichtbare Tests werden nicht in die Punktebewertung einbezogen.",
```

Do you think that simplicity in the code is a better trade-off than the current solution?